rotkehlchen/api/rest.py

```diff
@@ -997,20 +998,21 @@ def edit_ledger_action(self, action: LedgerAction) -> Response:
         result_dict = {'result': response['result'], 'message': response['message']}
         return api_response(process_result(result_dict), status_code=HTTPStatus.OK)
 
-    def delete_ledger_action(self, identifier: int) -> Response:
+    def delete_ledger_actions(self, identifiers: List[int]) -> Response:
         db = DBLedgerActions(self.rotkehlchen.data.db, self.rotkehlchen.msg_aggregator)
-        with self.rotkehlchen.data.db.user_write() as cursor:
-            error_msg = db.remove_ledger_action(cursor, identifier=identifier)
-        if error_msg is not None:
-            return api_response(wrap_in_fail_result(error_msg), status_code=HTTPStatus.CONFLICT)
-
-        # Success - return all ledger actions after the removal
-        response = self._get_ledger_actions(
-            filter_query=LedgerActionsFilterQuery.make(),
-            only_cache=True,
-        )
-        result_dict = {'result': response['result'], 'message': response['message']}
-        return api_response(process_result(result_dict), status_code=HTTPStatus.OK)
+        try:
+            with self.rotkehlchen.data.db.user_write() as cursor:
+                db.remove_ledger_actions(cursor, identifiers=identifiers)
+        except InputError as e:
+            return api_response(wrap_in_fail_result(str(e)), status_code=HTTPStatus.CONFLICT)
+        else:
+            # Success - return all ledger actions after the removal
+            response = self._get_ledger_actions(
+                filter_query=LedgerActionsFilterQuery.make(),
+                only_cache=True,
+            )
```

Comment:
We are not using pagination here and this can be a slow query. @lukicenturi do you use the ledger actions response when deleting a ledger action? Right now seems to return all without pagination

Reply:
Checking through the FE code, I don't think the result is being used. So I'm removing it.

Reply:
> We are not using pagination here and this can be a slow query. @lukicenturi do you use the ledger actions response when deleting a ledger action? Right now seems to return all without pagination

Although `only_cache=True` does not make it slow